### PR TITLE
Integration initialization

### PR DESCRIPTION
**Pull request by jnmrclmbsjse**
This PR includes:
1. Rest info endpoint
2. Connect to StoreKeeper button and action

### CODE REVIEW

**Comment by jnmrclmbsjse** [comment]
Task linked: [CU-8677hw2tf Woocomerce: Webshop integrations initialization flow](https://app.clickup.com/t/8677hw2tf)

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ConnectionTab.php [hunk 1]
```diff
@@ -22,13 +23,19 @@ class ConnectionTab extends AbstractTab
     use FormElementTrait;
 
     const DISCONNECT_ACTION = 'disconnect-action';
+    const CONNECT_ACTION = 'connect-action';
     const SAVE_ACTION = 'save-action';
 
+//    const INTEGRATIONS_HOST = 'https://integrations.storekeeper.software';
+    const INTEGRATIONS_HOST = 'https://junmar-accounts.code4.pizza';
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ConnectionTab.php:30]: can you make it 
```
if( !defined('STOREKEEPER_WOOCOMMERCE_INTEGRATIONS') ) {
define('STOREKEEPER_WOOCOMMERCE_INTEGRATIONS', 'https://integrations.storekeeper.software');
}
```

 in `storekeeper-woocommerce-b2c.php`  when we can overwrite it in `wp-config.php` easily

File: src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoEndpoint.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+            $response['account_name'] = $this->getAccountName();
+            $response['shop'] = $this->getShopDetails();
+            $response['webhook'] = $this->getWebhookDetails();
+        } catch (AccessDeniedException $exception) {
+            throw new WpRestException('Access Denied: '.$exception->getMessage(), 401, $exception);
+        } catch (GeneralException $exception) {
+            throw new WpRestException('General exception: '.$exception->getMessage(), 500, $exception);
+        } catch (\Throwable $exception) {
+            throw new WpRestException('Something went wrong', 500, $exception);
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoEndpoint.php:40]: maybe good to log the exeption using out logger, will save time debugging later

File: src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ConnectionTab.php (hunk 1 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/ConnectionTab.php]: Done.

File: src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoEndpoint.php (hunk 2 above)
Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoEndpoint.php:45]: Added logging.

**Pull request merged**